package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@terrazzo/monorepo",
   "private": true,
   "type": "module",
-  "packageManager": "pnpm@9.15.4",
+  "packageManager": "pnpm@10.6.2",
```

Comment:
10.6 adds support for more options in pnpm-workspace. I to configure the shellemulator option to run scripts.

Reply:
Nice! Yeah this can always be updated. Turborepo just requires a version specified but we can always be on latest